courses/advanced_exception_analysis/010_advanced_exception_analysis.rst

```diff
@@ -0,0 +1,452 @@
+*****************************
```

Comment:
Not sure where to comment it, but the section: Explicitly-Raised Exceptions is a little cluttered and tough to follow. Not sure if maybe if it's being rendered poorly on my side. Not a big issue at all, but would be nice if could be cleaned up.